packages/client/hmi-client/src/components/code/tera-directory.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<aside class="min-w-10rem max-w-20rem">
```

Comment:
Please use CSS. This is an abomination from hell, and should never be in the front-end with a ten foot pole.

Reply:
😆 I didn't know we were so against these classes!!

Reply:
Those classes are awful, it's like coding in the 90's. It's saying that CSS is a useless technology.

Reply:
i see i see, I sort of liked them cause it made it easy to add those simple styles instead of making a new class everytime(i.e flex).  Made the change though to use a class instead